packages/plugins/rubygem-foreman_ansible/rubygem-foreman_ansible.spec

```diff
@@ -37,7 +37,7 @@ BuildRequires: rubygem(foreman-tasks) >= 5.2.0
 BuildRequires: npm(@babel/core) >= 7.7.0
 BuildRequires: npm(@babel/core) < 8.0.0
 BuildRequires: npm(@theforeman/builder) >= 8.4.1
-BuildRequires: npm(@theforeman/builder) < 11.0.0
+BuildRequires: npm(@theforeman/builder) < 9.0.0
```

Comment:
This really needs to be fixed in the actual project. I started https://github.com/theforeman/foreman_ansible/pull/544.